src/dict/stardict.cc

```diff
@@ -352,9 +352,7 @@ class PowerWordDataProcessor
       QStringList sl;
       walkNode( doc.firstChild(), sl );
 
-      QStringListIterator itr( sl );
-      while ( itr.hasNext() ) {
-        QString s = itr.next();
+      for ( auto s : sl ) {
```

Comment:
This has to be a copy.

Because `translatePW` modifies `s`.

Note that QString is COW, so copying is ok but less ideal.